sample_challenge.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+PROMPT_COMMAND="$(pwd)/go-term-adventure $(pwd)/sample_challenge.gta" HISTFILE=$HOME/.advhistory bash
```

Comment:
@mrshu If this should be a `sample file` shouldn't this also contain option to specify --print?